tests/ui/ferrocene/compiler-arguments/upper_c_no-vectorize-loops/upper_c_no-vectorize-loops_no-space.rs

```diff
@@ -0,0 +1,9 @@
+// As other codegen options, no-vectorize-loops should also work without a space after -C.
+
+//@ check-pass
+//@ edition: 2021
```

Comment:
Do not specify `//@ edition: 2021`. Once we are done with the edition testing changes, adding that argument will imply the test is only supported on 2021, which is wrong.